pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -677,10 +714,11 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 				BackendAddressPool: &network.SubResource{
 					ID: to.StringPtr(lbBackendPoolID),
 				},
-				LoadDistribution: loadDistribution,
-				FrontendPort:     to.Int32Ptr(port.Port),
-				BackendPort:      to.Int32Ptr(port.Port),
-				EnableFloatingIP: to.BoolPtr(true),
+				LoadDistribution:     loadDistribution,
+				FrontendPort:         to.Int32Ptr(port.Port),
+				BackendPort:          to.Int32Ptr(port.Port),
+				EnableFloatingIP:     to.BoolPtr(true),
+				IdleTimeoutInMinutes: lbIdleTimeout,
```

Comment:
IdleTimeoutInMinutes only applies to TCP protocol. Could you also add a check and only set it when transportProto is TCP?

Reply:
Yep, also updated annotation name to clarify that.